### PR TITLE
Remi constrast specification

### PR DESCRIPTION
**Pull request by Remi-Gau**
Here is my attempt at refactoring the FFX and FFX_MVPA.

I have not yet 'tested' it because I don't have the data you used for MVPA: I would rather test on the same data set as you to start with. Just to make sure everything runs smoothly.

I have added the field `isMVPA` to the `opt` structure: this can be easily used as a "switch" to get different behaviour from `getFFXDir` or `BIDS_FFX`.

On thing that needs to be added is for the FFX analysis to be run on the right images: I think we mentioned changing `getPrefix` so it would get the slice time corrected and realigned images but not normalized images but I can't see this change at the moment. Am I missing something?

On that note, won't SPM throw an error if you try to use realigned but not resliced images for a FFX? Shouldn't our pipeline include the option of reslicing after realignement?

### CODE REVIEW

**Comment by Remi-Gau** [comment]
also pmCon seems like a good candidate for another unit test just to make sure it does what we want.

File: subfun/pmCon.m [hunk 1]
```diff
@@ -30,109 +30,107 @@
...
+                
+                % specify all the contrasts
+                
+                % first the contrasts to compute automatically against baseline
+                for iCon = 1:length(model.Steps{iStep}.AutoContrasts)
+                    
+                    con_counter = con_counter + 1;
+                    
+                    C = zeros(1,size(SPM.xX.X,2));
+
```
Comment (Remi-Gau) [inline subfun/pmCon.m]: Seems git diff is getting quite confused here. Might be better checking the whole file.

**Comment by Remi-Gau** [comment]
@mohmdrezk OK: I tried this on another data and this works for me now. Double check that this is fine for you too.

The problem we have now is that if we want to use the unnormalized data for MVPA we still need reslice the realigned images but since this is a different problem I suggest you send a PR from this branch onto the CPP repo and then we start working on fixing the preprocessing pipeline for the MVPA usecase.

**Comment by mohmdrezk** [comment]
> also pmCon seems like a good candidate for another unit test just to make sure it does what we want.

**Pull request merged**